docs/recipes.md

```diff
@@ -59,11 +59,15 @@ Guides, tips and strategies for getting the most out of your Foam workspace!
 
 ## Publish
 
-- Publish to [[github-pages]]
-- Publish to [[gitlab-pages]]
-- Publish your site with [[eleventy-and-netlify]]
-- Publish to [[azure-devops-wiki]]
-- Publish to [[vercel]]
+- Publish using official Foam template
```

Comment:
Perhaps "publish using **an** official Foam template" would be good here? Apologies for nit-picking!

Reply:
tricky one, right now we have "one" official foam template, but I don't think it's really up to date.. but we'll get to it soon :) for now I'd say let's move forward!